session1/challenge/client/src/js/components/presentational/TileErrorBoundary.js

```diff
@@ -1,6 +1,22 @@
 import React, { Component } from "react";
 import { DataTileStyled } from '../container/App.styled';
 
-const TileErrorBoundary = null;
+class TileErrorBoundary extends React.Component {
```

Comment:
you already have `Component` here, so you can avoid using `React.Component` :)

Reply:
Ok, i forgot to use **Component**, instead of **React.Component**, it was an oversight, but thank you for your comment.